journal/src/test/java/io/camunda/zeebe/journal/JournalReaderTest.java

```diff
@@ -15,33 +15,25 @@
 import io.camunda.zeebe.util.buffer.DirectBufferWriter;
 import java.io.File;
 import java.nio.charset.StandardCharsets;
-import java.nio.file.Path;
 import org.agrona.concurrent.UnsafeBuffer;
-import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.io.TempDir;
 
 final class JournalReaderTest {
 
   private static final int ENTRIES = 4;
+  @TempDir File directory;
 
   private final UnsafeBuffer data = new UnsafeBuffer("test".getBytes(StandardCharsets.UTF_8));
   private final BufferWriter recordDataWriter = new DirectBufferWriter().wrap(data);
-  private JournalReader reader;
-  private SegmentedJournal journal;
-
-  @BeforeEach
```

Comment:
:x: Correct me if I'm wrong, but can't we use `AfterEach` here to close the journal? Specifically, by adding it at the end of the test, if a test fails, the journal is not closed, right, as the line is never called? :thinking: